src/methods/delta/helpers/buildDeltaOrderData.ts

```diff
@@ -21,10 +21,26 @@ const SWAP_ORDER_EIP_712_TYPES = {
     { name: 'bridge', type: 'Bridge' },
   ],
   Bridge: [
-    { name: 'maxRelayerFee', type: 'uint256' },
-    { name: 'destinationChainId', type: 'uint256' },
-    { name: 'outputToken', type: 'address' },
-    { name: 'multiCallHandler', type: 'address' },
+    {
+      name: 'protocolSelector',
+      type: 'bytes4',
+    },
+    {
+      name: 'destinationChainId',
+      type: 'uint256',
+    },
+    {
+      name: 'outputToken',
+      type: 'address',
+    },
+    {
+      name: 'scalingFactor',
```

Comment:
what's this for? handling different decimals between the chains?

Reply:
yes, exactly